pythia/learning/tests/test_datasets.py

```diff
@@ -0,0 +1,134 @@
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pytest
+from astropy.io import fits
+from pythia.learning.datasets import AR_Dataset
+from pythia.learning.transforms import *
+from torchvision import transforms
+
+path = Path.cwd() / "learning/tests/test_data/"
```

Comment:
```suggestion
TEST_DATA = Path(__file__).parent / "test_data"
```
as a global variable, you should call it in capitals, and `path` is not too meaningful ;)